src/components/TopBar.js

```diff
@@ -230,143 +308,93 @@ export default _.flow(
+            titleIcon: 'library',
+            title: 'Terra Library',
+            onClick: () => this.setState({ openLibraryMenu: !openLibraryMenu }),
+            openMenu: openLibraryMenu,
+            subItems: [
+              h(DropDownSubItem, {
+                href: Nav.getLink('library-datasets'),
+                title: 'Data',
+                onClick: () => this.hideNav()
+              }),
+              h(DropDownSubItem, {
+                href: Nav.getLink('library-showcase'),
+                title: 'Showcase',
+                onClick: () => this.hideNav()
+              }),
+              h(DropDownSubItem, {
+                href: Nav.getLink('library-code'),
+                title: 'Tools',
+                onClick: () => this.hideNav()
+              })
+            ]
+          }),
           (trialState === 'Enabled') && enabledCredits,
           (trialState === 'Enrolled') && enrolledCredits,
           (trialState === 'Terminated') && terminatedCredits,
-          h(Clickable, {
-            style: { ...styles.nav.supportItem, marginTop: '15px' },
-            hover: { backgroundColor: colors.dark(0.55) },
-            onClick: () => contactUsActive.set(true)
-          }, [
-            div({ style: styles.nav.icon }, [
-              icon('envelope', { className: 'is-solid', size: 20 })
-            ]),
-            'Contact Us'
-          ]),
-          h(Clickable, {
-            style: styles.nav.supportItem,
-            as: 'a',
-            hover: { backgroundColor: colors.dark(0.55) },
-            href: 'https://support.terra.bio/hc/en-us',
-            ...Utils.newTabLinkProps,
-            onClick: () => this.hideNav()
-          }, [
-            div({ style: styles.nav.icon }, [
-              icon('help', {
-                className: 'is-solid',
-                size: 20
-              })
-            ]),
-            'Learn about Terra',
-            icon('pop-out', {
-              size: 12,
-              style: { marginLeft: '0.5rem' }
-            })
-          ]),
-          h(Clickable, {
-            style: styles.nav.supportItem,
-            as: 'a',
-            hover: { backgroundColor: colors.dark(0.55) },
-            href: 'https://support.terra.bio/hc/en-us/community/topics/360000500452',
-            ...Utils.newTabLinkProps,
-            onClick: () => this.hideNav()
-          }, [
-            div({ style: styles.nav.icon }, [
-              icon('bubble-exclamation', {
-                className: 'is-solid',
-                size: 20
+          h(BuildDropDownSection, {
+            titleIcon: 'help',
+            title: 'Terra Support',
+            onClick: () => this.setState({ openSupportMenu: !openSupportMenu }),
+            openMenu: openSupportMenu,
+            subItems: [
+              h(DropDownSubItem, {
+                href: 'https://support.terra.bio/hc/en-us',
+                title: 'How-to Guides',
+                onClick: () => this.hideNav(),
```

Comment:
I know it's always been like this but I'm finding it very awkward that the nav closes after opening these in a new window. It's like the nav hid away while I wasn't looking. I don't like losing that context with what I just clicked on.